static/css/custom.css

```diff
@@ -58,281 +86,181 @@ h2 {
-  padding: 33px 24px;
-}
-
-.card-wide:hover {
-  text-decoration: none;
-}
-
-.card-wide-content {
-  text-align: left;
-  margin: 0 16px;
-  flex: 1;
+  align-items: center;
+  transition: box-shadow 0.6s cubic-bezier(0.165, 0.84, 0.44, 1);
+  position: relative;
+  overflow: initial;
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.15);
+  border-radius: var(--ifm-global-radius);
 }
 
-.card-wide-content h4 {
-  margin: 0 0 8px 0;
-  font-size: 17px;
-  color: #000;
+[class*='home-card--'] {
+  padding: var(--space-s);
 }
 
-.card-wide-arrow {
+[class*='home-card--']::after {
+  content: '';
+  border-radius: var(--ifm-global-radius);
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
   height: 100%;
-  display: flex;
+  z-index: -1;
+  box-shadow: 0 3px 9px rgba(0, 0, 0, 0.25);
+  opacity: 0;
+  transition: opacity 0.3s ease-in-out;
 }
 
-.resources a {
-  color: dodgerblue;
-}
-
-.resourceContainer {
-  display: flex;
-  justify-content: space-between;
-  margin-bottom: 48px;
+[class*='home-card--']:hover::after {
+  opacity: 1;
 }
 
-.resourceContainer + p {
-  color: #a4a8b1;
+.home-card--basic {
+  max-width: 16em;
+  justify-content: center;
 }
 
-.resource {
-  flex: 1;
-  margin-left: 30px;
-  margin-right: 30px;
+.home-card--basic img {
+  width: 100%;
 }
 
-.resource:first-child {
-  margin-left: 0;
+.home-card--wide {
+  flex: 0 1 100%;
+  max-width: 30em;
 }
 
-.resource:last-child {
-  margin-right: 0;
+@media screen and (min-width: 900px) {
+  .home-card--wide {
+    flex: 0 1 calc(50% - 1em);
+  }
 }
 
-.resource p {
-  color: #636975;
+.home-card--wide {
+  gap: var(--space-s);
+  color: var(--ifm-color-secondary-lightest);
 }
 
-.metaTagContainer {
-  max-width: 900px !important;
+.home-card__logo {
+  max-width: 5em;
+  height: auto;
 }
 
-.metaTagContainer p {
-  color: #636975;
+.home-card__content h3 {
+  margin-bottom: var(--space-3xs);
 }
 
-.metaTagContainer span {
-  color: #636975;
+.home-card__content p {
+  font-size: var(--step--1);
+  margin-bottom: 0;
 }
 
-#paymentPointerForm {
+.home-cta {
   display: flex;
-  margin-bottom: 15px;
-}
-
-.paymentPointerInput {
-  height: 56px;
-  color: #a4a8b1;
-  background: #f5f6f8;
-  border-radius: 4px;
-  padding: 16px 24px;
-  font-size: 17px;
-  border: 0;
-  flex: 1;
-  font-family: $myOtherFont;
-}
-
-#generateButton {
-  height: 56px;
-  font-size: 16px;
-  color: #ffffff;
-  border-radius: 4px;
-  background: #0080ff;
-  margin-left: 16px;
-  padding-left: 26px;
-  padding-right: 26px;
-  cursor: pointer;
-  font-family: $myOtherFont;
+  align-items: center;
+  gap: var(--space-2xs);
+  margin-bottom: var(--space-s);
+  justify-content: center;
 }
 
-.metaTagOutput {
-  margin-top: 50px;
-  line-height: 2.5em;
+.home-cta p {
+  margin: 0;
 }
 
-#metaTag {
-  padding: 18px 50px 18px 24px;
-  display: block;
-  line-height: 2em;
-  position: relative;
-  background: #f7f8f9;
+.icon--merge {
+  width: 1em;
 }
 
+/* Link tag generator page styles */
```

Comment:
These will be dealt with in each respective page's refactor in future fast-follow PRs.